crates/cheatcodes/src/utils.rs

```diff
@@ -225,6 +225,29 @@ impl Cheatcode for copyStorageCall {
     }
 }
 
+impl Cheatcode for sortCall {
+    fn apply(&self, _state: &mut Cheatcodes) -> Result {
+        let Self { array } = self;
+
+        let mut sorted_values = array.clone();
+        sorted_values.sort();
+
+        Ok(sorted_values.abi_encode())
+    }
+}
+
+impl Cheatcode for shuffleCall {
+    fn apply(&self, _state: &mut Cheatcodes) -> Result {
+        let Self { array } = self;
+
+        let mut shuffled_values = array.clone();
+        let mut rng = thread_rng();
+        shuffled_values.shuffle(&mut rng);
```

Comment:
rng should be the one from the state: https://github.com/foundry-rs/foundry/blob/60adcb5c3a243b739282ae8c25adecb06c8b2625/crates/cheatcodes/src/inspector.rs#L1185